addons/payment/views/portal_templates.xml

```diff
@@ -16,6 +16,21 @@
         </xpath>
     </template>
 
+    <template id="portal_my_home_payment" name="Show Payment Method" customize_show="True" inherit_id="portal.portal_my_home" priority="60">
+        <xpath expr="//div[hasclass('o_portal_docs')]" position="before">
+            <t t-set="portal_client_category_enable" t-value="True"/>
+        </xpath>
+        <div id="portal_client_category" position="inside">
+            <t t-call="portal.portal_docs_entry">
+                <t t-set="icon" t-value="'/payment/static/img/payment-methods.svg'"/>
+                <t t-set="title">Payment method</t>
+                <t t-set="text">Add, remove or modify your payment methods</t>
+                <t t-set="url" t-value="'/my/payment_method'"/>
+                <t t-set="config_card" t-value="True"/>
+            </t>
+        </div>
```

Comment:
This should be shown under the same conditions as the `payment.pay_meth_link` template. If no token exists and no provider is enabled, it makes no sense to show this card.